locale/es/manageiq.po

```diff
@@ -26929,6 +26935,9 @@ msgstr "Credenciales de autenticación no válidas"
 msgid "Invalid button action"
 msgstr "Acción de botón no válida"
 
+msgid "Invalid button action!"
+msgstr "Acción de botón no válida."
```

Comment:
Just noticed this is missing the `!` (and since it's Spanish, the leading `¡`).  Again, doesn't matter cause it's going to go away.